armi/utils/codeTiming.py

```diff
@@ -420,9 +419,10 @@ def stop(self):
         cur_time = MasterTimer.time()
 
         if self._frozen:
-            return
+            return cur_time
 
-        if self.over_start:  # can't end the timer as it's over-started
+        if self.over_start:
+            # can't end the timer as it's over-started
             self.over_start -= 1
         elif self.isActive:
```

Comment:
Should this just be `else:`? Or a separate if statement?

Reply:
Oh, nope. That failed. 

It appears there are several more options than active and frozen.z

Now we know.